ocfweb/docs/docs/staff/startertasks.md

```diff
@@ -6,8 +6,8 @@ free to ask for help in our [Slack workspace](https://ocf.io/slack) or in person
 during [staff hours](https://ocf.io/staffhours)!
 
 Tasks marked with an asterisk (\*) require staff privileges. If you want to work
-on these but aren't on staff, let a current staffer know you’re working on
- starter tasks and we will add you.
+on these but aren't officially on staff yet, let a current staffer know you’re
```

Comment:
I think this should be clarified, "on staff" here means "in the ocfstaff LDAP group"

Reply:
I think it would be confusing. That's what "officially on staff" means.